config/tests/jobs/jobs_test.go

```diff
@@ -753,11 +753,6 @@ func checkScenarioArgs(jobName, imageName string, args []string) error {
 	}
 
 	if scenario == "" {
-		entry := jobName
-		if strings.HasPrefix(jobName, "pull-security-kubernetes") {
```

Comment:
Why did you remove this? Do we not ready the `entry` variable anymore?

Reply:
Yes, ```entry``` is no longer used.